contrib/nginx/template.py

```diff
@@ -0,0 +1,72 @@
+pkgname = "nginx"
+pkgver = "1.24.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--prefix=/etc/nginx",
+    "--sbin-path=/usr/bin/nginx",
+    "--modules-path=/usr/libexec/nginx",
+    "--conf-path=/etc/nginx/nginx.conf",
+    "--http-log-path=/var/log/nginx/access.log",
+    "--error-log-path=/var/log/nginx/error.log",
+    "--pid-path=/run/nginx.pid",
+    "--with-compat",
+    "--with-pcre",
+    "--user=_www",
+    "--group=_www",
+    "--http-client-body-temp-path=/var/tmp/nginx/client_body_temp",
+    "--http-fastcgi-temp-path=/var/tmp/nginx/fastcgi_temp",
+    "--http-proxy-temp-path=/var/tmp/nginx/proxy_temp",
+    "--http-scgi-temp-path=/var/tmp/nginx/scgi_temp",
+    "--http-uwsgi-temp-path=/var/tmp/nginx/uwsgi_temp",
+    "--with-http_v2_module",
+    "--with-http_addition_module",
+    "--with-http_auth_request_module",
+    "--with-http_dav_module",
+    "--with-http_flv_module",
+    "--with-http_gunzip_module",
+    "--with-http_gzip_static_module",
+    "--with-http_mp4_module",
+    "--with-http_random_index_module",
+    "--with-http_realip_module",
+    "--with-http_secure_link_module",
+    "--with-http_slice_module",
+    "--with-http_ssl_module",
+    "--with-http_stub_status_module",
+    "--with-http_sub_module",
+    "--without-mail_imap_module",
+    "--without-mail_pop3_module",
+    "--without-mail_smtp_module",
+    "--with-mail_ssl_module",
+    "--with-stream_realip_module",
+    "--with-stream_ssl_module",
+    "--with-stream_ssl_preread_module",
+    "--with-threads",
+    "--with-mail=dynamic",
+    "--with-stream=dynamic",
+]
+make_cmd = "bmake"
+make_dir = "."
+hostmakedepends = ["bmake"]
+makedepends = ["openssl-devel", "pcre2-devel", "zlib-devel"]
+pkgdesc = "Robust and small WWW server"
+maintainer = "Renato Botelho do Couto <renato@netgate.com>"
+license = "BSD-2-Clause"
+url = "https://nginx.org"
+source = f"https://nginx.org/download/{pkgname}-{pkgver}.tar.gz"
+sha256 = "77a2541637b92a621e3ee76776c8b7b40cf6d707e69ba53a940283e30ff2f55d"
+# Not available on nginx build tools
+options = ["!check"]
```

Comment:
it is, you can get it from https://hg.nginx.org/nginx-tests/archive/22f45bf99a9e etc

Reply:
from what i can tell it would also need perl-fcgi perl-protocol-websocket and uwsgi. the first two should be easy to package